images/php/Dockerfile.cli

```diff
@@ -46,9 +46,9 @@ COPY db-build.sh /bay
 COPY db-dump-sanitized.sh /bay
 COPY mtk /bay/mtk
 
-# Change worker pool from dynamic to static. Change default value to 24.
```

Comment:
Why are these lines being removed?

Reply:
Oh just noticed these to clean up - no fpm process in the cli container so we dont need the worker config here at all. Just cruft.

The same line still exists in the fpm image.